lib/basket.rb

```diff
@@ -0,0 +1,23 @@
+class Basket
+  attr_reader :summary
+  attr_accessor :items
+
+  def initialize
+    @items = []
+  end
+
+  def summary(items = unique_items)
```

Comment:
Really nice job with making the code provide a sophisticated view of the basket (items shown with their quantities).  To make methods like this easier to write, try to break them up into sub methods.  One way to do this is to make a list of the things the method needs to do (get a unique list of dishes, get the number of each dish ordered, turn the dish name and number to a nicely formatted string) etc and then write methods that do each thing.  You've started this nicely with `unique_items` - just carry on down this path.